### PR TITLE
Add group and path attributes

### PR DESCRIPTION
**Pull request by gctucker**
Add the `Node.group` and `Node.path` attributes when submitting nodes where applicable.

Depends on https://github.com/kernelci/kernelci-api/pull/167 and https://github.com/kernelci/kernelci-api/pull/168

### CODE REVIEW

File: config/runtime/kunit.jinja2 [hunk 1]
```diff
@@ -67,10 +67,12 @@ cd {src_path} && \
             'parent': parent['_id'],
             'status': 'complete',
             'revision': parent['revision'],
+            'group': parent['group'],
         }
         for node in child_nodes:
             base_node.update({
                 'name': node['name'],
+                'path': parent['path'] + [node['name']],
                 'result': node.get('result'),
```
Comment (JenySadadia) [inline config/runtime/kunit.jinja2:76]: How do we skip the particular test and why?
Need to add logic to set node result to `SKIP`.

Comment (gctucker) [reply config/runtime/kunit.jinja2:76]: Err, sorry?  I don't see what you mean.  This PR is about adding the `path` attribute, how is that related to the node result?

Comment (gctucker) [reply config/runtime/kunit.jinja2:76]: OK I see.  The "SKIP" result from KUnit is a key in the `RESULT_MAP` dictionary so any test that KUnit has skipped will have a result set to `None` in the data structure.

**Comment by gctucker** [comment]
@JenySadadia Does this look OK to you now?

**Comment by JenySadadia** [comment]
> @JenySadadia Does this look OK to you now?

Approved.

**Review by JenySadadia — APPROVED** [review]

**Pull request merged**